src/sass/_shame.scss

```diff
@@ -5,11 +5,12 @@
 
 // message while loading react app
 .loading-message {
-  text-align: center;
   padding: 2em 0;
+  text-align: center;
 }
 
-body .row {
+
+.row {
```

Comment:
@cehsu You can make a separate tech debt issue for making the qualifying element changes in that file if you'd like.